server/datastore/mysql/migrations/tables/20230927155121_CreateTableQueryReports.go

```diff
@@ -0,0 +1,35 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230927155121, Down_20230927155121)
+}
+
+func Up_20230927155121(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+		CREATE TABLE query_results (
+			id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+			query_id INT(10) UNSIGNED NOT NULL,
+			host_id INT(10) UNSIGNED NOT NULL,
+			osquery_version VARCHAR(50),
+			error TEXT COLLATE utf8mb4_unicode_ci DEFAULT NULL,
```

Comment:
TODO(lucas): We might not get the errors from osquery schedule results. Just tested this.